### ISSUE TITLE
Voidpets gone

### ISSUE THREAD

**Issue by Superredpanda** [voidpet/bugs#779]
Describe the bug below and provide your Voidpet username.

I was doing a grind for abandoned void pet and when i released 9 pets they were gone and I didn't get any idems for them either 

Username Superpanda

**Comment by benawad** [comment]
Does this happen every time you release pets or just this 1 time?

**Comment by Superredpanda** [comment]
It has happened this one time

**Comment by Superredpanda** [comment]
It has happened to me a few time now since the last comment

**Comment by benawad** [comment]
Do the wishes show up in your inventory if you refresh?

**Comment by Superredpanda** [comment]
No not at all

**Comment by benawad** [comment]
I'm assuming it only happens when you forget multiple pets at once, could you try recording it happening?

**Comment by Superredpanda** [comment]
Yes it does it has happened when I forget 3 pets and it did it,I'll try rn

**Comment by Superredpanda** [comment]
Now it don't want to do it, so I think it fixed its self, I tried to do it with 13 pets and it worked fine

**Comment by Superredpanda** [comment]
Is there any way I can get my pets back

**Comment by benawad** [comment]
Your pets are permanently gone, but I can refund your missing wishes. How many total pets went missing and what species?

**Comment by Superredpanda** [comment]
It was all total 50 sad wishes

**Comment by Superredpanda** [comment]
I recounted and it was 100 sad wished

**Comment by benawad** [comment]
Just sent you 100 sad wishes

**Comment by Superredpanda** [comment]
Okay thank you so much

**Comment by Superredpanda** [comment]
So you know the same thing keeps on happening.

**Comment by benawad** [comment]
It's hard to debug if it only happens sometimes. If you could pay attention to the times it occurs, maybe there is a factor thats different when it works vs doesn't? If you can catch it on video that is very helpful as well!

**Comment by Superredpanda** [comment]
I'll try my best, it seems to be doing it to me so far every time, I'll catch some pets and try to record it.

**Comment by Superredpanda** [comment]
I believe it has fixed itself because now it's working fine, I'm able to release 25 pets at once and no problem.

**Comment by Superredpanda** [comment]
It hasn't done it since the last comment, is there a way I can be refunded the wishes form when it wasn't

**Comment by benawad** [comment]
yeah, how many missing wishes?

**Comment by Superredpanda** [comment]
Anger - 2000 
Curiosity- 1000 
Grumpy- 500 
Lonely- roughly 5000 
Sad - 200 
Abandoned- roughly 6000 

Thank you so much

**Comment by benawad** [comment]
For that number of wishes, I would need to see a video of it not working.

**Comment by Superredpanda** [comment]
I wasn't able to get one, my computer doesnt let me screen record, can we do half of what I said

**Comment by Superredpanda** [comment]
I got a video of it were do you want me to send it ?

**Comment by benawad** [comment]
You can upload to GitHub

**Comment by Superredpanda** [comment]
okay , im sorry for the late answer, its not letting me upload it

**Comment by benawad** [comment]
upload to youtube and send a link
